docs/local_install_docker_compose.md

```diff
@@ -1,102 +1,139 @@
-# Local install of Substra (macOS; UNIX like system)
-
-This guide will help you to get Substra source code and to start a local instance with 2 organisations: owkin and chu-nantes.
-
-## Dependencies
-
-- docker: [Docker Desktop](https://www.docker.com/products/docker-desktop)
-- python 3 (recommended 3.6 or 3.7)
-  - it is recommended to use a virtual environment to install Substra packages (for instance [virtualenv](https://virtualenv.pypa.io/en/latest/))
-- yarn
-- redis server
-
-## Configuration
-
-- docker:
-  - on macOS, by default, docker has access only to the user directory.
+# Local install of Substra (macOS & Ubuntu/Debian)
+
```

Comment:
Would be maybe nice to explain there hat we encourage to use the skaffold setup instead of docker compose ?